gallery/wdio.conf.js

```diff
@@ -56,14 +57,17 @@ exports.config = {
     // and 30 processes will get spawned. The property handles how many capabilities
     // from the same test should run tests.
     //
-    maxInstances: 10,
+    maxInstances: 5,
     //
     // If you have trouble getting all important capabilities together, check out the
     // Sauce Labs platform configurator - a great tool to configure your capabilities:
     // https://docs.saucelabs.com/reference/platforms-configurator
     //
     capabilities: [{
-        browserName: 'chrome',
+      browserName: 'chrome',
+      'goog:chromeOptions': {
+        args: [ 'headless', 'font-render-hinting=none']
```

Comment:
Run in headless mode.  Without the `font-render-hinting` setting, the fonts come out substantially differently.

Reply:
If we run in headless mode will the mouse events in your tests above not work correctly?

Reply:
> If we run in headless mode will the mouse events in your tests above not work correctly?

They do seem to work correctly.  Headless mode seems to be able to simulate a mouse, and more importantly, each headless browser instance seems to simulate the mouse independently whereas with XVFB I think there's a single mouse globally which is why the XVFB approach couldn't successfully run mouse tests in parallel.